openhantek/src/mainwindow.h

```diff
@@ -37,6 +37,8 @@ public slots:
     // Central widgets
     DsoWidget *dsoWidget;
 
+    VoltageDock *voltageDock;
```

Comment:
Why is this necessary?

Reply:
Because otherwise QT cannot connect the signal from the Qdialog fished to the slot in the VoltageDock.